<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_minecraft">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>75</y>
     <width>381</width>
     <height>481</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>image/Minecraft.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_touhou7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>75</y>
     <width>381</width>
     <height>481</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>image/Touhou 7 Perfect Cherry Blossom.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_touhou6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>75</y>
     <width>381</width>
     <height>481</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>image/Touhou 6 Embodiment of Scarlet Devil.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_mahjong_soul">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>75</y>
     <width>381</width>
     <height>481</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>image/Mahjong Soul.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="buttonHorizontalLayout" stretch="0,0,0,0">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="button_minecraft">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Minecraft</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_touhou7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Touhou 7</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_touhou6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Touhou 6</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_mahjong_soul">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Mahjong Soul</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>label_mahjong_soul</zorder>
  <zorder>label_touhou6</zorder>
  <zorder>label_touhou7</zorder>
  <zorder>button_minecraft</zorder>
  <zorder>button_touhou7</zorder>
  <zorder>button_touhou6</zorder>
  <zorder>button_mahjong_soul</zorder>
  <zorder>label_minecraft</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
